### ISSUE TITLE
Resuming to map view crashes the app.

### ISSUE THREAD

**Issue by jraanamo** [eegeo/eegeo-example-app#51]
Open the main view (world map), switch to another application, return to main view.
Cloned from commit: 6e20e318890d33693e701593a1b8b32b18895fd1

```
06-27 14:49:38.926  4692  4716 E libEGL  : eglMakeCurrent:787 error 300d (EGL_BAD_SURFACE)
06-27 14:49:38.926  4692  4716 I Eegeo   : ASSERT: app/src/main/jni/./../../../../../platform//Android/AndroidSharedGlContext.cpp (24) 
06-27 14:49:39.301  4692  4717 I Eegeo   : uv_run took 842 milliseconds
06-27 14:49:39.301  4692  4717 I Eegeo   : OnWebResumeMessage: sleep duration is now 33 ms
06-27 14:49:39.355  3736  4319 D audio_hw_primary: disable_audio_route: usecase(1) reset and update mixer path: low-latency-playback
06-27 14:49:39.365  4893  7340 D GraphicsStats: Buffer count: 4
06-27 14:49:39.365  4893  9582 I WindowManager: WIN DEATH: Window{f1a6df9 u0 com.eegeo.mobileexampleapp/com.eegeo.mobileexampleapp.BackgroundThreadActivity}
06-27 14:49:39.365  4893  9582 W WindowManager: Force-removing child win Window{d7fce0e u0 SurfaceView - com.eegeo.mobileexampleapp/com.eegeo.mobileexampleapp.BackgroundThreadActivity} from container Window{f1a6df9 u0 com.eegeo.mobileexampleapp/com.eegeo.mobileexampleapp.BackgroundThreadActivity} 
```

**Comment by jonty-dawson** [comment]
Thanks - we have a fix for this under way, expect a commit soon

**Comment by tim-jenks** [comment]
Should be fixed by 7ec09d22caed7090d75280b196732d5a21b70e9d
